tests/javascript_tests/index.html

```diff
@@ -0,0 +1,77 @@
+<!--
+/**
+ *
+ * @author Yoshiki Vazquez Baeza
+ * @copyright Copyright 2013, The ili Project
+ * @credits Yoshiki Vazquez Baeza
+ * @license BSD
+ * @version 0.9.3-dev
+ * @maintainer Yoshiki Vazquez Baeza
+ * @email yoshiki89@gmail.com
+ * @status Development
+ *
+ */
+-->
+
+<!DOCTYPE html>
+<html>
+<head>
+  <!--
+      Note that all of the resources listed in this file assume the standard
+      base structure of ili. It is likely that this will not work properly
+      if it's not executed as suggested.
+  -->
+  <title>ili test suite</title>
+  <link rel="stylesheet" href="qunit/qunit-min.css" type="text/css" media="screen">
+
+  <!-- Testing framework requirements -->
+  <script src="qunit/qunit-min.js"></script>
+
+  <!-- Qunit depends on jQuery -->
+  <script src="../../js/lib/jquery-2.1.4.min.js"></script>
+
+  <!-- Libraries the code depends on -->
+
+  <!--Source files to test-->
+  <script src="../../js/ColorMaps.js"></script>
```

Comment:
This is fine as is, but just want to note that once we finish require-fying we are going to need to update this to use require.